.storybook/main.js

```diff
@@ -1,12 +1,17 @@
+import { dirname, join } from "path";
 const path = require("path");
 module.exports = {
   stories: ["../stories/**/*.stories.@(js|jsx|ts|tsx)"],
-  addons: ["@storybook/addon-essentials", "@storybook/addon-interactions"],
+  addons: [getAbsolutePath("@storybook/addon-essentials"), getAbsolutePath("@storybook/addon-interactions")],
 
   // framework: "@storybook/react-webpack5",
   // framework: "@storybook/react-vite",
-  framework: "storybook-react-parcel",
+  framework: getAbsolutePath("storybook-react-parcel"),
   // features: {
   //   storyStoreV7: false,
   // },
 };
+
+function getAbsolutePath(value) {
```

Comment:
Why was this necessary? Probably something with `package.json#exports`? But it did also work without this for me